packages/es-components/src/components/patterns/starRating/StarRatingExplanation.js

```diff
@@ -0,0 +1,269 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import styled from 'styled-components';
+import formatMessage from 'format-message';
+import { formatChildren } from 'format-message/react';
+import { Modal, ModalButtonContainer, Button, Table } from '../../../index';
+
+const HelpContent = styled(Modal.Body)`
+  ul {
+    display: block;
+    list-style-type: disc;
+    margin-bottom: 10px;
+    margin-left: 0;
+    margin-right: 0;
+    padding-left: 40px;
+    ul {
+      list-style-type: circle;
+    }
+  }
+`;
+
+const CautionRating = styled.div`
+  background-image: url(https://wtw-bdaim-cdn.azureedge.net/es-assets/es-assets-master/images/poor-performer-mask.svg);
+  background-size: 100% 100%;
+  height: 30px;
+  width: 150px;
+`;
+
+const StarRating = styled.div`
+  background-image: url(https://wtw-bdaim-cdn.azureedge.net/es-assets/es-assets-master/images/star-rating-mask.svg);
+  background-color: #ffc436;
+  height: 28px;
+`;
+
+const RatingTable = styled(Table)`
+  border: 1px solid black;
+  text-align: left;
+  margin: 15px 0px;
+  thead {
+    font-weight: bold;
+  }
+  th,
+  td {
+    border: 1px solid black;
+    padding: 8px;
+    vertical-align: middle;
+  }
+  td {
+    height: 63px;
+  }
+`;
+
+export default function StarRatingExplanation(props) {
+  return (
+    <Modal show={props.show} size="large" onHide={props.closeModal}>
+      <Modal.Header>{formatMessage('Plan Ratings')}</Modal.Header>
+      <HelpContent>
+        <p>
+          {formatMessage(
+            'The Overall Plan Rating combines scores, as rated by Medicare, for the types of services each plan offers.'
```

Comment:
Up until now we've been pretty content-agnostic. Should we pull this out, or change this explanation component to accept a render prop?

Reply:
This is a case where we want consistency in the content. This originally was in a different project but other teams want to use it as is without having to import a whole other set of dependencies when they already have this one. 

I asked if people wanted the entirety of it or just the actual ratings part and they said it was all helpful.

Reply:
I feel like we're muddying the water with this, but I can see how it would be a lot of content to make everyone reproduce.

Reply:
Sure, I get it. I don't know of a better way to share this and keep it consistent though. If you feel strongly, we can maybe think of something else.

Reply:
The only thing that comes to mind right now is setting the messages in a default object or render prop which can be overridden. There's a lot of text in various places, which makes this a little more difficult. We can go with it as-is for now and refactor later if we want.

We have a new Azure endpoint for assets, so the images in here need to be added to the newer internal one that the new Azure endpoint is using as its source:
https://github.mktp.io/extend-health/es-assets

new Azure endpoint:
https://bdaim-webexcdn-p.azureedge.net/es-assets/